system/jlib/jptree.ipp

```diff
@@ -242,59 +241,6 @@ struct AttrValue
 // Ideally the classes would be restructured to avoid this, but it would probably require AttrMap to move into PTree
```

Comment:
You can now delete
AM_NOCASE_FLAG, AM_NOCASE_MASK
just above this line in the header